app/controllers/capitals_controller.rb

```diff
@@ -36,7 +52,7 @@ def create
   def creation_service
     @creation_service ||= CapitalsCreationService.call(
       assessment_id: params[:assessment_id],
-      bank_accounts_attributes: input.dig(:liquid_capital, :bank_accounts),
+      bank_accounts_attributes: input[:liquid_capital],
```

Comment:
`input[:liquid_capital]` is used here but the expected param is `bank_accounts` (line 23)
I'm surprised no tests are failing.

Reply:
Good catch.  Commenting out the testing of the format of the response also meant that we weretn't testing how many records were created.  Fizxed now.